src/constants.js

```diff
@@ -54,7 +54,8 @@ export const SELF_SERVER_APP = {
     INVITATION_CODE: 'rc',
   },
   REGGIE: {
-    README: 'https://github.com/bcgov/reggie-web/blob/master/README-user-instructions.md',
+    README:
+      'https://developer.gov.bc.ca/Community-Enablers-and-Events/BC-Gov-Development-Community-Events',
```

Comment:
looks like this path is incorrect? 
It should be pointing towards the title of the source for the user instructions?
`Steps-to-join-Pathfinder-Rocket.Chat`

Reply:
lets double check prod to get the right link of course

Reply:
prod of developer.gov.bc.ca